dist/images/.gitignore

```diff
@@ -3,6 +3,7 @@ ovn-kube-util
 ovnkube
 ovnkube-trace
 ovnkube-identity
+ovnkube-observ
```

Comment:
This is not related.

Reply:
what do you want me to do?

Reply:
I mean, I am ok adding those to this PR since the PR itself is super small, if you think is not worth it to do the .gitignore at different PR I am fine, we can go with it.